frontend/src/components/elements/PlotlyChart/PlotlyChart.tsx

```diff
@@ -88,4 +93,32 @@ export function PlotlyChart({
   }
 }
 
+function layoutWithThemeDefaults(layout: any, theme: Theme): any {
+  const { colors, genericFonts, inSidebar } = theme
+  const themeBg = inSidebar ? colors.sidebarBg : colors.bgColor
```

Comment:
Once my sidebar change comes through, we wont need to do this! I've made the bgColor for the sidebarTheme into colors.bgColor and just switch up the bgColor for the sidebarTheme. Assuming this will merge in first so making a note for me to update this once it is!

Reply:
:+1: 

Note that there's a similar change in applying themes to `VegaLiteCharts` too so this will need to be changed in 2 places.